core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiService.scala

```diff
@@ -135,6 +135,13 @@ trait EntityApiService extends UserInfoDirectives {
             }
           }
         } ~
+        path("workspaces" / Segment / Segment / "entityType" / "rename") { (workspaceNamespace, workspaceName) =>
+          post {
+            entity(as[EntityTypeRename]) { rename =>
+              complete { entityServiceConstructor(userInfo).renameEntityType(WorkspaceName(workspaceNamespace, workspaceName), rename.oldName, rename.newName).map(_ => StatusCodes.NoContent) }
+            }
+          }
+        } ~
```

Comment:
style question: is it worth passing the entire `EntityTypeRename` object as an argument into `renameEntityType`, instead of passing the two Strings? That would allow us in the near future to add fields to the `EntityTypeRename` definition for merge options without having to change `renameEntityType`'s signature.

Reply:
Seems reasonable, done.